tests/mark/check_cluster_size.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from ducktape.mark import parametrize, matrix, ignore
+from ducktape.mark._mark import MarkedFunctionExpander
+from ducktape.mark import resource
```

Comment:
Why not just import `cluster_size` instead of repeating `resource.cluster_size` throughout this file